README.md

```diff
@@ -89,26 +87,26 @@ Give an example
 
 ## Deployment
 
-Add additional notes about how to deploy this on a live system
-
+Visit (https://papnews.herokuapp.com/) for deployed app
```

Comment:
remove ( ) ?

Reply:
I would keep it, but format it differently in Markdown
```
Visit [the deployed app](https://papnews.herokuapp.com/).
```

Reply:
What @tochman said